packages/flutter/lib/src/widgets/table.dart

```diff
@@ -292,21 +292,34 @@ class _TableElement extends RenderObjectElement {
   @override
```

Comment:
The comment on line 287 about slots not being used seems outdated?

Reply:
Removed.